### PR TITLE
[RFC] Use container infrastructure.

### PR DESCRIPTION
**Pull request by fwalch**
Travis added a notice on top of build logs ("This job ran on our legacy infrastructure. Please read our docs on how to upgrade"), so I looked into changing this to the container infrastructure (again).

Waiting for https://github.com/travis-ci/apt-package-whitelist/issues/192, then it should work. `sync-mirrors` is the only failing job right now (build: https://travis-ci.org/fwalch/bot-ci/builds/72141810).

### CODE REVIEW

**Comment by fwalch** [comment]
`bzr-fastimport` has been whitelisted on Travis, the build now works: https://travis-ci.org/fwalch/bot-ci/builds/72141810

Removed a test commit, marking RFC.

File: ci/sync-mirrors.sh [hunk 1]
```diff
@@ -26,9 +26,9 @@ sync_bzr_mirror() {
   commit_subtree MIRROR
 }
 
-is_ci_build && {
+#is_ci_build && {
   install_git_bzr
-}
+#}
```
Comment (justinmk) [inline ci/sync-mirrors.sh]: why not check `is_ci_build` ?

**Comment by justinmk** [comment]
LGTM other than minor question. Thanks for doing this!

File: ci/sync-mirrors.sh (hunk 1 above)
Comment (fwalch) [inline ci/sync-mirrors.sh]: Oops, commented this for local testing, will fix it.

**Comment by fwalch** [comment]
Will merge tomorrow to not trigger Coverity again today.

**Pull request closed without merging**